drills/deploy/README.md

```diff
@@ -0,0 +1,160 @@
+## Purpose of Deploy.rb
+This document for who using Mina gem at https://github.com/mina-deploy/mina
+
+This document describle how to use 1 deploy.rb file for multible environment.
+
+##Install mina gem
+Open gem file, add mina gem in development group
+
+    group :development do
+      gem 'mina'
+    end
+    
+Then, open terminal, install it
+
+    $ bundle install
+
+After installation is finished, run
+
+    $ mina init
+    Created config/deploy.rb.
+
+##Initialize Amazon web services for deploy.
+Contact adminstrator to setting up deploy environment on AWS.
+
+#Setting mina for environment
+    
+    $ mina setup to=your_app_environment
+
+E.g: mina setting for staging environment
+    
+    $ mina setup to=staging
+
+##Setting common config for both Production environment and Staging environment.
+Open config/deploy.rb
+
+    app               = 'your_app_name'
+    production_domain = 'your_app_production_domain'
+    staging_domain    = 'your_app_staging_domain'
+    user              = 'ec2-user'
```

Comment:
This is default user in AWS. Heroman said that when create a new user, he have to change many thing. It will hard for both administrator an developer. So he recommence using default user for all projects.